src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.services.AreaCalculator;
+import core.basesyntax.services.ColorSupplier;
+import core.basesyntax.services.Drawer;
+
+public class Circle extends Figure implements AreaCalculator, Drawer {
+    protected double radius;
+
+    public Circle(double radius) {
+        this.radius = radius;
+        super.setColor(draw());
+    }

```

Comment:
```suggestion
    public Circle(String color, double radius) {
        super(color)
        this.radius = radius;
    }

```

let's accept color as constructor parameter in all figures classes